src/components/keyboard-nav.js

```diff
@@ -0,0 +1,93 @@
+import * as _ from 'lodash/fp'
+import { useRef } from 'react'
+import { h } from 'react-hyperscript-helpers'
+import { toIndexPairs } from 'src/libs/utils'
```

Comment:
As a rule for utils.js, we import `* as Utils` and then call those as `Utils.foo`